src/@next/pages/CheckoutPage/subpages/CheckoutPaymentSubpage.tsx

```diff
@@ -158,6 +165,7 @@ const CheckoutPaymentSubpageWithRef: RefForwardingComponent<
       errors = dataError?.error.extraInfo.userInputErrors;
     }
     if (errors) {
+      changeSubmitProgress(false);
```

Comment:
Why multiple `changeSubmitProgress(false);` instead of one at the end of function?

Reply:
Because another forms is going to be submitted and thus submit progress is still in progress. If there are errors, the next forms are not submitted, so then submit progress is set to false.